### ISSUE TITLE
[BUG] <title>

### ISSUE THREAD

**Issue by kiiriitsugu** [ldtteam/minecolonies#8758]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Are you using the latest MineColonies Verison?

- [X] I am running the latest **alpha** version of MineColonies for my Minecraft version.

### Did you check on the Wiki? or ask on Discord?

- [X] I checked the MineColonies Wiki and made sure my issue is not covered there. Or I was sent from discord to open an issue here.

### Minecraft Version

1.18

### MineColonies Version

1.0.1105-ALPHA

### Structurize Version

1.0.424-ALPHA

### Related Mods and their Versions

Launcher set to latest forge version
0.0.62-ALPHA BlockUi
1.0.55-ALPHA Domum

### Current Behavior

The latest minecraft laucher update made it so my modpack doesn't run on curseforge launcher anymore. I Switched over to ATLauncher, and copied over the files in my instance folder to bring over my save and mod configs.

After loading in ATLauncher, i was no longer the owner of my colony.  However when i interacted with the townhall it stated i already owned a colony, When using the list comand it only had a single colony(ID 1). when trying to delete it, it said the colony could not be found.  

While the townhall claimed i owned a colony, /mc whoami said this:
![image](https://user-images.githubusercontent.com/117217252/200662859-b67204af-dc4f-4cc4-90a8-3d635bfbdcaf.png)

Tried to use the comands to set abandoned, then set owner again. the colony was sucessfully set to abandoned but set owner 
didn't work. everything was the same. 

After closing and reloading the modpack, things were like this : 
![image](https://user-images.githubusercontent.com/117217252/200663502-b35726f6-4324-4d82-9de8-54aca15a0dc8.png)

tryed to set owner again, and check colony info

![image](https://user-images.githubusercontent.com/117217252/200664218-7d774f68-949b-4438-af1e-08f3beb79bcf.png)

it now says i am the owner in the colony list command and whoami.
![image](https://user-images.githubusercontent.com/117217252/200664947-c8dcbd8a-b5d3-47c4-93fa-9a080341a890.png)

but not on the townhall

![image](https://user-images.githubusercontent.com/117217252/200665068-d33b9450-59be-4c16-828b-6b6ff803e3e6.png)

it acts as if there is a diferent colony nearby, with the same name. but list only shows one.

![image](https://user-images.githubusercontent.com/117217252/200665285-5be2c49c-ca8a-4eed-b6ca-5c3d478e81fc.png).

/mc colony loadBackup 1 tried to /mc colony loadBackup 1. 

The results where identical until closed and reopened the game.
 
 After opening again, the townhall interaction changed to this:
 
![image](https://user-images.githubusercontent.com/117217252/200673750-286ca8ae-247a-4281-97f1-d512b598203e.png)

But when i try to click create new colony it still acts like there is another colony nearby, and won't let me do it.

I can break blocks as if it was my colony, but i cannot interact with colonists or hut blocks.

### Expected Behavior

After using the set owner command, being shown as the colony's owner in the townhall and being able to interact with things properly.

### Reproduction Steps

1. Create a modpack instance on curseforge launcher, create a save and a colony.
2. Copy over modpack instance folder  from curseforge launcher to ATLauncer. (haven't tried different launchers)
3. Try interacting with your colony, and check if you still own it.
4. Try using the command to set yourself as the owner of the colony.

### Logs

https://gist.github.com/kiiriitsugu/1ad1fe9ec1d96eb5d3022ddc43c6dbd8

### Anything else?

I'm playing life in the village 3 modpack (https://www.curseforge.com/minecraft/modpacks/life-in-the-village-3).

Things were working properly in curseforge untill this new minecraft launcher update.
I had copied instances inside curseforge, added more mods and loaded back into my save in the new instance, and didn't run into this issue. (after copying i was not the owner of the colony, but the set owner command worked properly in that scenario).

**Comment by Raycoms** [comment]
It seems like some chunk data went missing, can you try the /mc colony claim command to claim your colony area?

**Comment by uecasm** [comment]
Try updating the curseforge launcher, they said they fixed it.

**Comment by kiiriitsugu** [comment]
I was able to load the game in curseforge, seems like they did fix the issue with the launcher, and after opening my save, i was surprised to find out the same error is happening on curseforge too. seems like it had nothing to do with transfering from curseforge to AT. maybe something that occurred with this minecraft launcher update? i'm lost now

 /mc colony claim did not work either

**Comment by Raycoms** [comment]
Is this server in offline mode?

**Comment by kiiriitsugu** [comment]
I was able to figure out the issue. it was incompatibility with this mod i added :
 https://www.curseforge.com/minecraft/mc-mods/mcmt-multithreading.
 
 Since i played with it for a while before it happened and the issue coincided with switching from curseforge to atlauncher i didn't think of it at first.
 
 The error persisted when removing the mod too, but after removing it and rolling back to a few backups i had on curse, it was fixed and didn't happen again.
 
 My guess is the behavior is inconsistent and happens depending on how the treads are separated by the mod at launch.
 May be good to add that mod to a list of incompatibilities.
 
 ( and answering raycoms, was playing mostly offline but i was turning on lan with cheats enabled to use the commands.)
 
 Sorry for the inconvenience.
